packages/next/src/server/next-server.ts

```diff
@@ -1685,11 +1692,19 @@ export default class NextNodeServer extends BaseServer {
   }
 
   public getRequestHandler(): NodeRequestHandler {
+    const handler = this.makeRequestHandler()
+    if (this.serverOptions.experimentalTestMode) {
+      return wrapRequestHandler(handler)
```

Comment:
Same reason as https://github.com/vercel/next.js/pull/52520/files#r1265229370
```suggestion
      const { wrapRequestHandler } = require('./test')
      return wrapRequestHandler(handler)
```